src/site/content/en/metrics/inp/index.md

```diff
@@ -63,7 +63,7 @@ An interaction's latency consists of the single longest [duration](https://w3c.g
 For more details on how INP is measured, read the ["What's in an interaction?"](#whats-in-an-interaction) section.
 {% endAside %}
 
-## What's a "good" INP value?
+### What's a good INP value?
```

Comment:
```suggestion
### What is a good INP score?
```

For full consistency. Since these heading links get used a lot it'd be good to make them all the same.